symplyphysics/core/vectors/arithmetics.py

```diff
@@ -64,6 +64,13 @@ def add_cartesian_vectors(vector_left: Vector, vector_right: Vector) -> Vector:
     return Vector(result, vector_left.coordinate_system)
 
 
+def add_cartesian_vectors(vector: Vector, *vectors: Vector) -> Vector:
```

Comment:
Use single list of vectors as argument. Update `subtract` function to match the signature.

Reply:
You want it to be so that `subtract_cartesian_vectors(a, b, c)` should yield `a - b - c`?

Reply:
Yes.